### DISCUSSION TITLE
Can I bundle only one/some node dependencies (and not all as with --bundle-node-modules)?

### DISCUSSION THREAD

**Discussion by jlipps** [parcel-bundler/parcel#6082] [Q&A] [answered]
Hi everyone! I'm building a set of electron apps which share some code and components. I appreciate that Parcel doesn't bundle Node modules when the target is Electron. However, in my case, I have several apps which want to use "raw" (untranspiled, with CSS files alongside) components from a shared library.

It would be great if I could somehow direct Parcel to bundle only *this* node module. Otherwise I'm not sure how to hook into Parcel's systems that bundle JS and CSS and get them to target my shared library code.

Any thoughts or ideas?

**Comment by mischnic** [comment, marked as answer]
Parcel 2 supports this: https://v2.parceljs.org/configuration/package-json/#includenodemodules

**Reply by jlipps** [reply]
Thanks, I'll give upgrading a go!
